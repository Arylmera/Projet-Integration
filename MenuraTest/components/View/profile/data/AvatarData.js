/**
 * data pour les avatars
 */
let data = [
   {
      id: '1',
      img: require('../../../../assets/avatars/avatar1.png'),
   },
   {
      id: '2',
      img: require('../../../../assets/avatars/avatar2.png'),
   },
   {
      id: '3',
      img: require('../../../../assets/avatars/avatar3.png'),
   },
   {
      id: '4',
      img: require('../../../../assets/avatars/avatar4.png'),
   },
   {
      id: '5',
      img: require('../../../../assets/avatars/avatar5.png'),
   },
   {
      id: '6',
      img: require('../../../../assets/avatars/avatar6.png'),
   },
   {
      id: '7',
      img: require('../../../../assets/avatars/avatar7.png'),
   },
   {
      id: '8',
      img: require('../../../../assets/avatars/avatar8.png'),
   },
   {
      id: '9',
      img: require('../../../../assets/avatars/avatar9.png'),
   },
   {
      id: '10',
      img: require('../../../../assets/avatars/avatar10.png'),
   },
   {
      id: '11',
      img: require('../../../../assets/avatars/avatar11.png'),
   },
   {
      id: '12',
      img: require('../../../../assets/avatars/avatar12.png'),
   },
   {
      id: '13',
      img: require('../../../../assets/avatars/avatar13.png'),
   },
   {
      id: '14',
      img: require('../../../../assets/avatars/avatar14.png'),
   },
   {
      id: '15',
      img: require('../../../../assets/avatars/avatar15.png'),
   },
   {
      id: '16',
      img: require('../../../../assets/avatars/avatar16.png'),
   },
   {
      id: '17',
      img: require('../../../../assets/avatars/avatar17.png'),
   },
   {
      id: '18',
      img: require('../../../../assets/avatars/avatar18.png'),
   },
   {
      id: '19',
      img: require('../../../../assets/avatars/avatar19.png'),
   },
   {
      id: '20',
      img: require('../../../../assets/avatars/avatar20.png'),
   },
   {
      id: '21',
      img: require('../../../../assets/avatars/avatar21.png'),
   },
   {
      id: '22',
      img: require('../../../../assets/avatars/avatar22.png'),
   },
   {
      id: '23',
      img: require('../../../../assets/avatars/avatar23.png'),
   },
   {
      id: '24',
      img: require('../../../../assets/avatars/avatar24.png'),
   },
   {
      id: '25',
      img: require('../../../../assets/avatars/avatar25.png'),
   },
   {
      id: '26',
      img: require('../../../../assets/avatars/avatar26.png'),
   },
   {
      id: '27',
      img: require('../../../../assets/avatars/avatar27.png'),
   },
   {
      id: '28',
      img: require('../../../../assets/avatars/avatar28.png'),
   },
   {
      id: '29',
      img: require('../../../../assets/avatars/avatar29.png'),
   },
   {
      id: '30',
      img: require('../../../../assets/avatars/avatar30.png'),
   },
   {
      id: '31',
      img: require('../../../../assets/avatars/avatar31.png'),
   },
   {
      id: '32',
      img: require('../../../../assets/avatars/avatar32.png'),
   },
];
export default data;
