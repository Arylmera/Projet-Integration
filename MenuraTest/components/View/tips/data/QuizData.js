/**
 * data pour le quizz
 */
export default data = [


    {
        id: '1',
        reponse1: 'mésange',
        reponse2: 'hibou',
        reponse3: 'pic vert',
        reponse4: 'bruant jaune',
        reponseCorrecte: 'mésange',
        image: require('../../../../assets/quizz/mesange.jpg'),
    },
    {
        id: '2',
        reponse1: 'canard',
        reponse2: 'hibou',
        reponse3: 'moineau',
        reponse4: 'bruant jaune',
        reponseCorrecte: 'moineau',
        image: require('../../../../assets/quizz/moineau.jpg'),
    },
    {
        id: '3',
        reponse1: 'mésange',
        reponse2: 'mouette',
        reponse3: 'pic vert',
        reponse4: 'bruant jaune',
        reponseCorrecte: 'mouette',
        image: require('../../../../assets/quizz/mouette.jpg'),
    },
    {
        id: '4',
        reponse1: 'mésange',
        reponse2: 'hibou',
        reponse3: 'pic vert',
        reponse4: 'bruant jaune',
        reponseCorrecte: 'pic vert',
        image: require('../../../../assets/quizz/picvert.jpg'),
    },
    {
        id: '5',
        reponse1: 'mésange',
        reponse2: 'hibou',
        reponse3: 'bargerousse',
        reponse4: 'bruant jaune',
        reponseCorrecte: 'bargerousse',
        image: require('../../../../assets/quizz/bargerousse.jpg'),
    },
    {
        id: '6',
        reponse1: 'tarier',
        reponse2: 'hibou',
        reponse3: 'pic vert',
        reponse4: 'bruant jaune',
        reponseCorrecte: 'tarier',
        image: require('../../../../assets/quizz/tarier.jpg'),
    },
    {
        id: '7',
        reponse1: 'grand duc',
        reponse2: 'hibou',
        reponse3: 'pic vert',
        reponse4: 'bruant jaune',
        reponseCorrecte: 'grand duc',
        image: require('../../../../assets/quizz/grandduc.jpg'),
    },
    {
        id: '8',
        reponse1: 'mésange',
        reponse2: 'bruant jaune',
        reponse3: 'pic vert',
        reponse4: 'macreuse',
        reponseCorrecte: 'macreuse',
        image: require('../../../../assets/quizz/MACREUSE.jpg'),
    },
    {
        id: '9',
        reponse1: 'mouette',
        reponse2: 'hibou',
        reponse3: 'verdier',
        reponse4: 'macreuse',
        reponseCorrecte: 'verdier',
        image: require('../../../../assets/quizz/VERDIER.jpg'),
    },
    {
        id: '10',
        reponse1: 'pie bavarde',
        reponse2: 'verdier',
        reponse3: 'macreuse',
        reponse4: 'hibou',
        reponseCorrecte: 'pie bavarde',
        image: require('../../../../assets/quizz/PIEBAVARDE.jpg'),
    },

]
