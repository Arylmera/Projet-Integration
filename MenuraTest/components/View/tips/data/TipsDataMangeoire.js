

export default data = [


    {
        id: '1',
        title: 'Mangeoires brouette',
        description: 'Ils sont conçus pour pouvoir facilement le déplacer à l\'intérieur d\'une cabanne ou chalet de jardin',
        image: require('../../../../assets/images/Mangeoires/Mangeoire1_2.png'),
        image_Plan: require('../../../../assets/images/Mangeoires/Mangeoire1_3.png'),
    },
    {
        id: '2',
        title: 'Mangeoires plateau',
        description: 'La mangeoire à plateaux multiples permet aux oiseaux de retrouver leur nourriture sur différents paliers',
        image: require('../../../../assets/images/Mangeoires/Mangeoire4_1.png'),
        image_Plan: require('../../../../assets/images/Mangeoires/Mangeoire4_2.png'),
    },
    {
        id: '3',
        title: 'Mangeoire à étages',
        description: 'La partie haute pour les petites espèces, la partie basse pour les oiseaux de taille moyenne.',
        image: require('../../../../assets/images/Mangeoires/Mangeoire5_1.png'),
        image_Plan: require('../../../../assets/images/Mangeoires/Mangeoire5_2.png'),
    },
    {
        id: '4',
        title: 'Mangeoires couvertes',
        description: 'La mangeoire couvercle permet à l\'oiseau de rester à l\'abris fasse aux autres prédateurs',
        image: require('../../../../assets/images/Mangeoires/Mangeoire6_1.png'),
        image_Plan: require('../../../../assets/images/Mangeoires/Mangeoire6_2.png'),
    },
    {
        id: '5',
        title: 'Mangeoire murale',
        description: 'La mangeoire murale permet facilement de l\'accrocher à n\'importe quel mur ou surface de l\'extérieur de votre maison',
        image: require('../../../../assets/images/Mangeoires/Mangeoire7_1.png'),
        image_Plan: require('../../../../assets/images/Mangeoires/Mangeoire7_2.png'),
    },

]
