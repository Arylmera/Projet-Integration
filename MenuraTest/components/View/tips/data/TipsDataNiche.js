/**
 * data niches
 */
export default data = [


    {
        id: '1',
        title: 'Nichoir Boîte à lettres',
        description: 'C\'est le nichoir le plus commun. Le plus facile à construire aussi. Il convient à un grand nombre d\'espèces, en particulier aux mésanges, sittelles, etc...',
        description_Plan: 'Dimensions : voir ci-dessus "Dimensions optimales" (12 possibilités).\n' +
            'Pour la Mésange boréale, le fond du nichoir (type "Mésange bleue") doit être garni de sciure ou de copeaux.',
        image: require('../../../../assets/images/Niches/Niche1_1.png'),
        image_Plan: require('../../../../assets/images/Niches/Niche1_2.png'),
    },
    {
        id: '2',
        title: 'Nichoir toit à deux pans',
        description: 'C\'est une variante du nichoire à boite à lettre avec un toit à deux pans. Il convient à un grand nombre d\'espèces, en particulier aux mésanges, sittelles, etc...',
        description_Plan: 'Dimensions : voir ci-dessus "Dimensions optimales" (12 possibilités).\n' +
            'Pour la Mésange boréale, le fond du nichoir (type "Mésange bleue") doit être garni de sciure ou de copeaux.',
        image: require('../../../../assets/images/Niches/Niche2_1.png'),
        image_Plan: require('../../../../assets/images/Niches/Niche2_2.png'),
    },
    {
        id: '3',
        title: 'Nichoir style maisonette ',
        description: 'C\'est une autre variante du nichoire à boite à lettre avec un toit à deux pans. Il convient à un grand nombre d\'espèces, en particulier aux mésanges, sittelles, etc...',
        description_Plan: 'Dimensions : voir ci-dessus "Dimensions optimales" (12 possibilités).\n' +
            'Pour la Mésange boréale, le fond du nichoir (type "Mésange bleue") doit être garni de sciure ou de copeaux.',
        image: require('../../../../assets/images/Niches/Niche3_1.png'),
        image_Plan: require('../../../../assets/images/Niches/Niche3_2.png'),
    },
    {
        id: '4',
        title: 'Nichoir troglodyte ',
        description: 'Un nichoir idéal pour le troglodyte (mais peut accueillir mésanges, sittelle, etc...)',
        description_Plan: 'Dimensions : voir ci-dessus "Dimensions optimales" (12 possibilités).\n' +
            'Pour la Mésange boréale, le fond du nichoir (type "Mésange bleue") doit être garni de sciure ou de copeaux.',
        image: require('../../../../assets/images/Niches/Niche5_1.png'),
        image_Plan: require('../../../../assets/images/Niches/Niche5_2.png'),
    },
    {
        id: '5',
        title: 'Nichoir "boîte à lettres"',
        description: 'Un nichoir "boîte à lettres" avec ouverture sur le côté',
        description_Plan: 'Dimensions : voir ci-dessus "Dimensions optimales" (12 possibilités).\n' +
            'Pour la Mésange boréale, le fond du nichoir (type "Mésange bleue") doit être garni de sciure ou de copeaux.',
        image: require('../../../../assets/images/Niches/Niche6_1.png'),
        image_Plan: require('../../../../assets/images/Niches/Niche6_2.png'),
    },
    {
        id: '6',
        title: 'Nichoir "Zuern"',
        description: 'Le nichoir "Zuern", dit aussi "nichoir anti-prédateur". Très utilisé en Amérique du Nord.',
        description_Plan: 'Dimensions : voir ci-dessus "Dimensions optimales" (12 possibilités).\n' +
            'Pour la Mésange boréale, le fond du nichoir (type "Mésange bleue") doit être garni de sciure ou de copeaux.',
        image: require('../../../../assets/images/Niches/Niche7_1.png'),
        image_Plan: require('../../../../assets/images/Niches/Niche7_2.png'),
    },
    {
        id: '7',
        title: 'Nichoir semi-ouvert',
        description: 'Convient au Rougegorge, Rougequeue noir, Gobemouche gris, Bergeronnette grise...',
        description_Plan: 'Dimensions : voir la page " dimensions des nichoirs" (2 possibilités)',
        image: require('../../../../assets/images/Niches/Niche8_1.png'),
        image_Plan: require('../../../../assets/images/Niches/Niche8_2.png'),
    },
    {
        id: '8',
        title: 'Nichoir semi-ouvert',
        description: 'Convient au Rougegorge, Rougequeue noir, Gobemouche gris, Bergeronnette grise...',
        description_Plan: 'Dimensions : voir la page " dimensions des nichoirs" (2 possibilités)',
        image: require('../../../../assets/images/Niches/Niche9_1.png'),
        image_Plan: require('../../../../assets/images/Niches/Niche9_2.png'),
    },



]
