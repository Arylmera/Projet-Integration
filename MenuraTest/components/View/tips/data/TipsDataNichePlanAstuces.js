/**
 * data plan niches
 */
let data = [
   {
      id: '1',
      description:
         "Fond décalé : Si les côtés dépassent le plancher, la pluie pourrira d'abord le bas des parois et le fond sera protégé.",
      image: require('../../../../assets/images/NichoireAstuces/Astuce1.png'),
   },
   {
      id: '2',

      description:
         'Un fond posé sur deux tasseaux (et non cloué) facilite grandement le nettoyage annuel.',
      image: require('../../../../assets/images/NichoireAstuces/Astuce2.png'),
   },
   {
      id: '3',
      description:
         "Percer un trou d'envol obliquement sur le plan vertical minimise la pénètration de la pluie.",
      image: require('../../../../assets/images/NichoireAstuces/Astuce3.png'),
   },
   {
      id: '4',
      description:
         "Des clous plantés avec des orientations diverses évitent les risques d'arrachement.",
      image: require('../../../../assets/images/NichoireAstuces/Astuce4.png'),
   },
   {
      id: '5',
      description:
         'Une rainure sous le toit du nichoir sert de "goutte d\'eau" et empêche les infiltrations.',
      image: require('../../../../assets/images/NichoireAstuces/Astuce5.png'),
   },
   {
      id: '6',
      description:
         "Ecraser d'un coup de marteau la pointe du clou évite l'éclatement du bois.",
      image: require('../../../../assets/images/NichoireAstuces/Astuce6.png'),
   },
];
export default data;
