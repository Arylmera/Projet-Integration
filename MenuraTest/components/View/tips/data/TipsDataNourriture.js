'use strict'

export default data = [


    {
        id: '1',
        titre: 'Tournesol :',
        description: 'Graine très appréciée de la plupart des oiseaux des jardins, notamment des mésanges. Les moineaux la mangent peu s\'il disposent d\'autre nourriture.',
        image: require('../../../../assets/images/Nourritures/Nourriture1.jpg'),
    },
    {
        id: '2',
        titre: 'Graines de chardons :',
        description: 'Mangées par de nombreux oiseaux, particulièrement les fringilles (Pinsons, chardonnerets, verdiers, etc...).',
        image: require('../../../../assets/images/Nourritures/Nourriture2.jpg'),
    },
    {
        id: '3',
        titre: 'Blé, avoine, millet, maïs, chenevis, orge :',
        description: 'Attirent également beaucoup d\'oiseaux. Le chenevis est apprécié de toutes les espèces. Les moineaux et les bruants aiment l\'avoine, mais les mésange ne la consomme que faute de mieux. Le millet est surtout mangé par les moineaux.',
        image: require('../../../../assets/images/Nourritures/Nourriture3.jpg'),
    },
    {
        id: '4',
        titre: 'Plantes :',
        description: 'On peut aussi récolter ou réserver : les épis de plantain, les tiges d\'oseille sauvage, les pépins de pommes et de poires, de pastèque, de courges, les noyaux écrasés de cerises et de prunes, les gaines de rumex, d\'orties, de centaurée, de cardère, de séneçon.',
        image: require('../../../../assets/images/Nourritures/Nourriture4.jpg'),
    },
    {
        id: '5',
        titre: 'Baies :',
        description: 'On peut en récolter à l\'automne et les entreposer dans un endroit sec. Au moment de la récolte, pensez aux fauvettes qui en sont friandes et laissez en sur les arbustes. Ce sont les baies de sureau, genévrier, sorbier, aubépine.',
        image: require('../../../../assets/images/Nourritures/Nourriture5.jpg'),
    },
    {
        id: '6',
        titre: 'Fruits secs :',
        description: 'Noisettes débarassées de leur coquille, elles peuvent être placées dans une mangeoire. Noix, amendes décortiquées, la noix de coco ne doit pas être donnée séchée.',
        image: require('../../../../assets/images/Nourritures/Nourriture6.jpg'),
    },
    {
        id: '7',
        titre: 'Pommes ou poires blètes :',
        description: 'Jetées au sol, sous un arbre de préférence afin d\'offir aux oiseaux un refuge en cas de danger. Elles attirent merles, étourneaux, grives, verdiers, etc...',
        image: require('../../../../assets/images/Nourritures/Nourriture7.jpg'),
    },
    {
        id: '8',
        titre: 'Cacahuètes :',
        description: 'Non grillées et non salées ! Avec leur enveloppe, se suspendent aux branches. Débarassées de leur enveloppe, sont à déposer dans une mangeoire.',
        image: require('../../../../assets/images/Nourritures/Nourriture8.jpg'),
    },
    {
        id: '9',
        titre: 'Cônes de pin :',
        description: 'Jetés à terre ou suspendus.',
        image: require('../../../../assets/images/Nourritures/Nourriture9.jpg'),
    },
    {
        id: '10',
        titre: 'Mélange pour oiseaux :',
        description: 'On préfèrera les graines pour oiseaux "du ciel". Les mélanges pour oiseaux de volière sont aussi appréciés par les oiseaux sauvages. Dans un cas comme dans l\'autre, chaque espèce d\'oiseau ayant sa préférence, une bonne partie des graines risque d\'être rejetée de la mangeoire.',
        image: require('../../../../assets/images/Nourritures/Nourriture10.jpg'),
    },


]
