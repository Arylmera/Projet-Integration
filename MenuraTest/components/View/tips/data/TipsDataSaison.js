export default data = [
    {
        id: '1',
        saison: 'Automne',
        tips:
            'Dès l’arrivée des premiers froids, la nourriture commence à se raréfier pour les oiseaux.\n' +
            '\n' +
            'Apportez de la nourriture régulièrement pour compenser la raréfaction des ressources alimentaires.',
        image: require('../../../../assets/images/Saisons/Automne.png'),
        description_nourriture: 'L’automne est la saison où les oiseaux font des réserves pour l’hiver. Durant cette saison, il est plus difficile pour eux de trouver de la nourriture bien que leur corps demande justement de l’énergie suffisante pour rester à bonne température. S’il fait donc très froid en automne ou en hiver, vous pouvez les aider en les nourrissant.',
        description_abris:'Il est important que les oiseaux ne craignent pas de devenir de la nourriture de chats ou de rapaces pendant qu’ils se nourrissent ! C’est pourquoi il convient d’installer des abris près de la nourriture. Ne nettoyez pas non plus le jardin impeccablement mais laissez plutôt quelques feuilles mortes et des pots et panier vides. Autre conseil pratique : placer une grille ou une vannerie sur la nourriture pour en bloquer l’accès aux corbeaux et aux mouettes. Les petits oiseaux pourront ainsi déguster leur nourriture en toute tranquillité.'
    },
    {
        id: '2',
        saison: 'Hiver',
        tips:
            'Le gel et la neige limitent l’accès des oiseaux à l’eau et à la nourriture.\n' +
            '\n' +
            'Proposez de la nourriture telle que les graines ou/et des boules de graisse pour aider les oiseaux à résister à la période hivernale.\n' +
            '\n' +
            'Installez une baignoire à oiseaux et veillez à ce que l’eau ne soit pas gelée et toujours propre, changez l’eau régulièrement.',
        image: require('../../../../assets/images/Saisons/Hiver.png'),
        description_nourriture: 'L’hiver, c’est la saison durant laquelle il est conseillé de nourrir les oiseaux dans son jardin ou sur son balcon. Même lors des hivers assez doux, il est utile de les aider car leurs ressources alimentaires « naturelles » (graines laissées sur pied dans les cultures, baies dans les arbustes, etc.) ont tendance à diminuer à cause des techniques agricoles modernes. Ce sera ainsi l’occasion d’observer des espèces qui, le reste de l’année, ne s’approchent pas des maisons ou sont discrètes (grives, troglodytes, mésanges, etc.).',
        description_abris:'Il est important que les oiseaux ne craignent pas de devenir de la nourriture de chats ou de rapaces pendant qu’ils se nourrissent ! C’est pourquoi il convient d’installer des abris près de la nourriture. Ne nettoyez pas non plus le jardin impeccablement mais laissez plutôt quelques feuilles mortes et des pots et panier vides. Autre conseil pratique : placer une grille ou une vannerie sur la nourriture pour en bloquer l’accès aux corbeaux et aux mouettes. Les petits oiseaux pourront ainsi déguster leur nourriture en toute tranquillité.'
    },
    {
        id: '3',
        saison: 'Printemps',
        tips:
            'Il y a 4 fois plus d’oiseaux qu’en hiver, donc les besoins en alimentation augmentent. C’est aussi le début de la période de nidification.\n' +
            '\n' +
            'Remplissez régulièrement les mangeoires de nourriture.\n' +
            '\n' +
            'Préparez des nichoirs pour offrir aux oiseaux un lieu adapté pour couver et élever leurs petits.',
        image: require('../../../../assets/images/Saisons/Printemps.png'),
        description_nourriture: 'Le printemps, c\'est la saison durant laquelle les oiseaux du jardin sont très occupés à construire leur nid. Ils cherchent bien entendu de la nourriture pour leurs petits. À cette saison, les parents apprécient les mélanges de graines. À cette époque, la nourriture doit contenir davantage de calcium et de protéines, par exemple des vers de farine ou des coquilles d’œufs finement écrasées. Mieux vaut dès lors ne plus leur donner de boules de graisse ni de cacahuètes.',
        description_abris:'Il est important que les oiseaux ne craignent pas de devenir de la nourriture de chats ou de rapaces pendant qu’ils se nourrissent ! C’est pourquoi il convient d’installer des abris près de la nourriture. Ne nettoyez pas non plus le jardin impeccablement mais laissez plutôt quelques feuilles mortes et des pots et panier vides. Autre conseil pratique : placer une grille ou une vannerie sur la nourriture pour en bloquer l’accès aux corbeaux et aux mouettes. Les petits oiseaux pourront ainsi déguster leur nourriture en toute tranquillité.'
    },
    {
        id: '4',
        saison: 'Été',
        tips:
            'La période de nidification se poursuit pour aboutir à la naissance des petits, qui vont avoir besoin d’une alimentation spécifique pour leur croissance.\n' +
            '\n' +
            'De plus, avec la chaleur estivale, l’accès à l’eau pure, vitale pour l’oiseau, est de plus en plus difficile.\n' +
            '\n' +
            'Apportez une alimentation complémentaire aux oiseaux qui n’ont pas assez de temps pour collecter la nourriture pour leurs petits et pour eux.\n' +
            '\n' +
            'Installez une baignoire pour permettre aux oiseaux de se désaltérer et de se rafraîchir. Vous pouvez également mettre de l’eau dans une brique ou dans des couvercles de confitures…\n' +
            '\n',
        image: require('../../../../assets/images/Saisons/ETE.png'),
        description_nourriture: 'L\'été, c\'est la saison durant laquelle les oiseaux cherchent eux-mêmes les vers et les insectes dont ils ont besoin. Aidez-les en mettant dans votre jardin des plantes qui attirent les insectes. En arrosant votre pelouse, vous attirez des vers de terre.',
        description_abris:'Il est important que les oiseaux ne craignent pas de devenir de la nourriture de chats ou de rapaces pendant qu’ils se nourrissent ! C’est pourquoi il convient d’installer des abris près de la nourriture. Ne nettoyez pas non plus le jardin impeccablement mais laissez plutôt quelques feuilles mortes et des pots et panier vides. Autre conseil pratique : placer une grille ou une vannerie sur la nourriture pour en bloquer l’accès aux corbeaux et aux mouettes. Les petits oiseaux pourront ainsi déguster leur nourriture en toute tranquillité.'
    },
]
