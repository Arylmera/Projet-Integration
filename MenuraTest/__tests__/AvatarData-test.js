import 'react-native';
import data from '../components/View/profile/data/AvatarData';

test('quizz data test', () => {
   expect(data).toMatchSnapshot();
   expect(data).toHaveLength(32);
});

for (let i in data) {
   test(
      'data' +
         i +
         ' should have properties (id, response 1 to 4, reponseCorrecte, image)',
      () => {
         expect(data[i]).toHaveProperty('id');
         expect(data[i]).toHaveProperty('img');
      },
   );
}

test('item returns 3 firsts', () => {
   const item1 = data[0];
   const item2 = data[1];
   const item3 = data[2];
   const item = jest.fn((item) => item.id);
   item(item1);
   item(item2);
   item(item3);
   expect(item).toHaveLastReturnedWith('3');
   expect(item).toHaveBeenCalledTimes(3);
});

test('item data has id 1 and matches as an object', () => {
   const id1 = {
      id: '1',
      img: require('../assets/avatars/avatar1.png'),
   };
   expect(data[0]).toMatchObject(id1);
});
