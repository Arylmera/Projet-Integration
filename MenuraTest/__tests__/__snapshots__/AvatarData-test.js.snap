// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`quizz data test 1`] = `
Array [
  Object {
    "id": "1",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar1.png",
    },
  },
  Object {
    "id": "2",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar2.png",
    },
  },
  Object {
    "id": "3",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar3.png",
    },
  },
  Object {
    "id": "4",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar4.png",
    },
  },
  Object {
    "id": "5",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar5.png",
    },
  },
  Object {
    "id": "6",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar6.png",
    },
  },
  Object {
    "id": "7",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar7.png",
    },
  },
  Object {
    "id": "8",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar8.png",
    },
  },
  Object {
    "id": "9",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar9.png",
    },
  },
  Object {
    "id": "10",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar10.png",
    },
  },
  Object {
    "id": "11",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar11.png",
    },
  },
  Object {
    "id": "12",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar12.png",
    },
  },
  Object {
    "id": "13",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar13.png",
    },
  },
  Object {
    "id": "14",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar14.png",
    },
  },
  Object {
    "id": "15",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar15.png",
    },
  },
  Object {
    "id": "16",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar16.png",
    },
  },
  Object {
    "id": "17",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar17.png",
    },
  },
  Object {
    "id": "18",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar18.png",
    },
  },
  Object {
    "id": "19",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar19.png",
    },
  },
  Object {
    "id": "20",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar20.png",
    },
  },
  Object {
    "id": "21",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar21.png",
    },
  },
  Object {
    "id": "22",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar22.png",
    },
  },
  Object {
    "id": "23",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar23.png",
    },
  },
  Object {
    "id": "24",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar24.png",
    },
  },
  Object {
    "id": "25",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar25.png",
    },
  },
  Object {
    "id": "26",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar26.png",
    },
  },
  Object {
    "id": "27",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar27.png",
    },
  },
  Object {
    "id": "28",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar28.png",
    },
  },
  Object {
    "id": "29",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar29.png",
    },
  },
  Object {
    "id": "30",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar30.png",
    },
  },
  Object {
    "id": "31",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar31.png",
    },
  },
  Object {
    "id": "32",
    "img": Object {
      "testUri": "../../../assets/avatars/avatar32.png",
    },
  },
]
`;
